tests/security/yast2_apparmor/manually_add_profile.pm

```diff
@@ -103,13 +105,21 @@ sub run {
     send_key_until_needlematch("AppArmor-Inactive-local-profile", "tab", 2, 2);
     # Check "Use Profile"
     send_key "alt-u";
-    mouse_click("right");
     assert_screen("AppArmor-Scan-system-log");
     # Exit "yast2 apparmor"
     wait_screen_change { send_key "alt-f" };
 
     # Exit x11 and turn to console
+    # Close the yast2 apparmor window
+    save_screenshot;
     send_key "alt-f4";
+    wait_still_screen(5);
+
+    # Close the second xterm window
+    save_screenshot;
+    send_key "alt-f4";
+    wait_still_screen(5);
```

Comment:
Can we remove this? 
since `assert_screen` takes 60s there

Reply:
Sure, thanks.